release-notes/5.2/5.2.0-preview5.md

```diff
@@ -0,0 +1,73 @@
+# Release Notes
+
+## [Preview Release 5.2.0-preview5.YY365.#] - 2024-01-24
+
+### Added
+
+- Added .NET 8 Support from .NET 7 [#2230](https://github.com/dotnet/SqlClient/pull/2230)
+
+### Changed
+
+- Improved by avoiding boxing of SqlGuid in SqlBuffer [#2306](https://github.com/dotnet/SqlClient/pull/2306)
+- Improved TdsParserStateObject by reducing redundancy [#2254](https://github.com/dotnet/SqlClient/pull/2254)
+
+### Fixed
+
+- Fixed wrong dependency for netstandard2.0 [#2305](https://github.com/dotnet/SqlClient/pull/2305)
+- Fixed connection to unsubscribe from any provider before returning it to the connection pool [#2301](https://github.com/dotnet/SqlClient/pull/2301)
+- Fixed Microsoft.IdentityModel.JsonWebTokens version [#2290](https://github.com/dotnet/SqlClient/pull/2290)
+- Fixed SQL authentication check during debug mode [#2288](https://github.com/dotnet/SqlClient/pull/2288)
+- Fixed InvalidCastException when reading an always encrypted date column [#2275](https://github.com/dotnet/SqlClient/pull/2275)
+
+## Target Platform Support
+
+- .NET Framework 4.6.2+ (Windows x86, Windows x64)
+- .NET 6.0+ (Windows x86, Windows x64, Windows ARM64, Windows ARM, Linux, macOS)
+- .NET Standard 2.0+ (Windows x86, Windows x64, Windows ARM64, Windows ARM, Linux, macOS)
+
+### Dependencies
+
+#### .NET Framework
+
+- Microsoft.Data.SqlClient.SNI 5.2.0-preview1.23340.1 //TODO
+- Azure.Identity 1.10.3
+- Microsoft.Identity.Client 4.56.0
+- Microsoft.IdentityModel.JsonWebTokens 6.24.0
+- Microsoft.IdentityModel.Protocols.OpenIdConnect 6.24.0
+- System.Buffers 4.5.1
+- System.Configuration.ConfigurationManager 6.0.1
+- System.Runtime.InteropServices.RuntimeInformation 4.3.0
+- System.Text.Encoding.Web 6.0.0
+
+#### .NET
```

Comment:
Should probably be a .NET 6 and .NET 8 section now

For .NET 8 it is:

- Microsoft.Data.SqlClient.SNI 5.2.0-preview1.23340.1
- Azure.Identity 1.10.3
- Microsoft.Identity.Client 4.56.0
- Microsoft.IdentityModel.Protocols.OpenIdConnect 6.24.0
- Microsoft.IdentityModel.JsonWebTokens 6.24.0
- Microsoft.SqlServer.Server 1.0.0
- System.Configuration.ConfigurationManager 8.0.0
- System.Diagnostics.DiagnosticSource 8.0.0
- System.Runtime.Caching 8.0.0

For .NET 6 it is:

- Microsoft.Data.SqlClient.SNI 5.2.0-preview1.23340.1
- Azure.Identity 1.10.3
- Microsoft.Identity.Client 4.56.0
- Microsoft.IdentityModel.Protocols.OpenIdConnect 6.24.0
- Microsoft.IdentityModel.JsonWebTokens 6.24.0
- Microsoft.SqlServer.Server 1.0.0
- System.Configuration.ConfigurationManager 6.0.1
- System.Diagnostics.DiagnosticSource 6.0.1
- System.Runtime.Caching 6.0.0

Reply:
Leaving this un-resolved for now, need to talk to the team to see if they prefer this method or the one that is currently committed. But thank-you for all your feedback.

Reply:
Hi Eryk, leaving a few more commits on updates (you may want to wait), but we have decided to take your approach. Cheers.